frontend/src/pages/AllRunsList.tsx

```diff
@@ -103,6 +116,40 @@ class AllRunsList extends Page<{}, AllRunsListState> {
       this.props.history.push(RoutePage.NEW_RUN + searchString);
     }
   }
+
+  private async _archiveDialogClosed(confirmed: boolean): Promise<void> {
```

Comment:
This should get moved somewhere it can be shared

Reply:
I'm thinking about adding another `PageActions` file that contains all the logic that runs when clicking common page buttons. This might add a bit of abstraction to the reasoning, but will remove a lot of code duplication. WDYT?

Reply:
Yes, it'd be under /lib.
The good thing about a dedicated actions file is that it can reduce duplication across other actions too. We have 4 `cloneRun` implementations, for example, 3 `compareRun`s, 3 `newExperiment`s.. etc.